src/models/EventResult.ts

```diff
@@ -0,0 +1,6 @@
+import { SimpleEvent } from "./SimpleEvent";
+
+export interface EventResult {
+    readonly month: string
```

Comment:
We already have the start and end date for the event, I don't think grouping by month gives any useful information.

Reply:
@gigobyte Sorry for the late reply. I don't think we should get rid of the grouping completely. Maybe as an optional parameter? I think it gives useful "information" or better "structure".

Original Issue #102

Reply:
Maybe we can have the month as a number then? `new Date().getMonth()` returns a number so I'm assuming people working with the API will want to compare dates and months when using this endpoint.

Reply:
@gigobyte Added month as number. Also added it as an optional parameter for filtering the results for a specific month.

Reply:
Cool, can you resolve the merge conflict before I merge your contributions to master?

Reply:
@gigobyte Merge done. Had to do some additional fixes caused by the added eventId.

Reply:
@emha Cool, thank you, I'll look into the code soon.